src/main/starlark/builtins_bzl/common/cc/action_names.bzl

```diff
@@ -52,6 +52,7 @@ ASSEMBLE_ACTION_NAME = "assemble"
 PREPROCESS_ASSEMBLE_ACTION_NAME = "preprocess-assemble"
 
 LLVM_COV = "llvm-cov"
+DWP = "dwp"
```

Comment:
Could you add a comment to the DWP line, and while you are at it, also add one for LLVM_COV? I think it can be just "Name of the coverage action".